src/cockpit/389-console/src/lib/database/databaseTables.jsx

```diff
@@ -917,8 +921,277 @@ class PwpTable extends React.Component {
+                    <div className="ds-lower-field">
+                        <ArrowRightIcon /> {sort.attrs.vlvsort[0]} ({indexState})
+                    </div>
+                    <div>
+                        <Button
+                            className="ds-left-margin"
+                            onClick={() => {
+                                this.props.deleteSortFunc(row.attrs['cn'][0], sort.attrs.vlvsort[0]);
+                            }}
+                            id={row.attrs['cn'][0]}
+                            icon={<TrashAltIcon />}
+                            variant="link"
+                        >
+                            Delete
+                        </Button>
+                    </div>
+                </GridItem>
+            );
+        });
+
+        return (
+            <Grid>
+                <GridItem className="ds-label" span={2}>
+                    Search Base:
+                </GridItem>
+                <GridItem span={10}>
+                    {row.attrs.vlvbase[0]}
+                </GridItem>
+                <GridItem className="ds-label" span={2}>
+                    Search Filter:
+                </GridItem>
+                <GridItem span={10}>
+                    {row.attrs.vlvfilter[0]}
+                </GridItem>
+                <GridItem className="ds-label" span={2}>
+                    Scope:
+                </GridItem>
+                <GridItem span={10}>
+                    {this.getScopeKey(row.attrs.vlvscope[0])}
+                </GridItem>
+                <GridItem className="ds-label" span={12}>
+                    Sort Indexes:
+                </GridItem>
+                <div className="ds-margin-top ds-indent">
+                    {sort_indexes}
+                </div>
+                <GridItem className="ds-label" span={1}>
+                    <Button
+                        className="ds-margin-top"
+                        onClick={() => {
+                            this.props.addSortFunc(row.attrs['cn'][0]);
+                        }}
+                        variant="primary"
+                    >
+                        Create Sort Index
+                    </Button>
+                </GridItem>
+            </Grid>
+        );
+    }
+
+    componentDidMount() {
+        let rows = [];
+        let columns = this.state.columns;
+        let count = 0;
+
+        for (let row of this.props.rows) {
+            rows.push(
+                {
+                    isOpen: false,
+                    cells: [row.attrs['cn'][0], row.attrs.vlvbase[0]],
+                },
+                {
+                    parent: count,
+                    fullWidth: true,
+                    cells: [{ title: this.getExpandedRow(row) }]
+                },
+            );
+            count += 2;
+        }
+        if (rows.length == 0) {
+            rows = [{cells: ['No VLV Indexes']}];
```

Comment:
I wonder if it's the only way to implement this part... Because currently, it's possible to click on 'three dots' and you may try to 'Delete "No VLV Indexes"'. If it will be possible to hide the three dots when it's "No VLV Indexes" - it will be better.

But I understand that the tool may be clunky so it's a minor and feel free to waive it.

Reply:
If there are no vlv indexes then the table is not even displayed .  So I guess this code is dead actually.

Reply:
Weird the table disappeared for me, but on a refresh the empty table is there with the "dots".  The check for "rows" was incomplete.  I fixed it so now the dots are not present.